tests/VersionTest.php

```diff
@@ -368,6 +368,11 @@ public function test_it_can_compare_two_versions(): void
         $this->assertEquals(1, SemVer\Version::compare(new SemVer\Version('v1.0.2'), new SemVer\Version('v1.0.1')));
     }
 
+    public function test_it_is_correctly_json_serializable(): void
+    {
+        $this->assertEquals('1.3.37', new SemVer\Version('v1.3.37'));
```

Comment:
We should assert that a `Version` object is an instance of `JsonSerializable`. Also, it's preferable to call `jsonSerialize()` directly to ensure we're explicitly testing that method. I'm afraid the test as-written is incidentally testing the `__toString()` method.

```suggestion
        $version = new SemVer\Version('v1.3.37');

        $this->assertInstanceOf(JsonSerializable::class, $version);
        $this->assertEquals('1.3.37', $version->jsonSerialize());
```